contrib/osx/README.md

```diff
@@ -16,14 +16,21 @@ This assumes that the Xcode + Xcode Command Line tools (and thus git) are alread
 
 The above ensures that you pull in the zbar, secp256k1, and other submodules.
 
-## 2. Use the provided script to begin building.
+## 2. Make sure coreutils is installed
+
+With [brew](https://brew.sh) installed, run
```

Comment:
Two things:

1.  I believe brew is terrible and merits to die as a project.  Please also add MacPorts instructions while we are here:

`sudo port install coreutils`

2. Also pyenv is not needed for building this project -- the build script automatically installs pyenv for you. Please delete that line.

Reply:
I tried to run this  on Big Sur and for some reason the script failed with something like "pyenv is not installed, aborting...". 

This is why I added this line. Please confirm one more time that you don't want this line and I'll remove it :)

Reply:
Holy cow! That was NOT possible before AFAIK.  That's awesome.  TIL.

Reply:
See -- this is why I don't use brew. :)

Well -- then can we clarify that's a brew-specific thing?  Not everybody uses brew.  I certainly don't -- Josh Elliithorpe doesn't.. there are plenty of people  not using it...

Maybe make the `pyenn` line a brew-specific: e.g. "If on brew do this...", "If on macports do this...".

Reply:
Added